src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -171,22 +174,23 @@ public static EventTitle parseEventTitle(String eventTitle) throws ParseExceptio
     }
 
     /**
-     * Parses a {@code String Date} into a {@code Date}. {@code isFutureDateAllowed} is true
-     * if a date after the current date is allowed, false otherwise.
+     * Parses a {@code String Date} into a {@code Date}.
+     * {@code isFutureDateAllowed} is trueif a date after the current date is allowed, false otherwise.
```

Comment:
I think we can shorten the description for this:

```suggestion
     * The {@code isFutureDateAllowed} flag controls whether or not a date after the current date is allowed.
```